### DISCUSSION TITLE
Parent theme when mocking theme

### DISCUSSION THREAD

**Discussion by LoVas2** [keycloakify/keycloakify#83] [Q&A] [answered]
Hello, 
How can I change the parent theme during hot reload mode ?
Actually it's  base theme and I want the Keycloak one. 
Thx

theme.properties in KC : 
parent=keycloak
import=common/keycloak

**Comment by garronej** [comment, marked as answer]
Hi @LoVas2,  
It doesn’t work like this in keyclaokify.  The Keycloak theme has been rewritten in react from scratch.  
You have two options for customization:  
1) Just change the CSS:  https://github.com/garronej/keycloakify-demo-app
2) Write the components yourself:  https://github.com/garronej/keycloakify-demo-app/tree/look_and_feel  

Best,
